src/ResourceManager/AzureBatch/Commands.Batch/Models.Generated/PSTaskFailureInformation.cs

```diff
@@ -27,17 +27,16 @@ namespace Microsoft.Azure.Commands.Batch.Models
     using System.Collections;
     using System.Collections.Generic;
     using Microsoft.Azure.Batch;
-
-
-    [Obsolete("SchedulingError will be removed in a future version and replaced with FailureInformation")]
```

Comment:
Need an example of usage of the new type and how this has changed in the upcoming-breakign changes document

Reply:
The type itself is used the same way (it's structured the same, just renamed). I will clarify in upcoming-breaking-changes though.